main-app/app/scripts/controller.js

```diff
@@ -1,10 +1,19 @@
 
 (function () {
     'use strict';
-    angular.module('tombola.snapCardGame')
+    angular.module('Tombola.SnapCardGame')
+
+        .controller('SnapCardController',['$scope','GameModel','DeckService',function ($scope, GameModel, DeckService){
```

Comment:
spaces after commas here as well please, again partially done but not consistent